frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -120,60 +142,49 @@ export function FeatureFlag(): JSX.Element {
                     requiredMark={false}
                     scrollToFirstError
                 >
```

Comment:
The page-level actions should be opposite the header to balance things out. I think the core problem is when the history tab is shown in an editing context. Ideally, the view for exiting a feature flag doesn't have these tabs at all. 

It might be out of scope for this effort, but I do have a redrawn feature flag creation page at https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=5400%3A35869. 

![create ff](https://user-images.githubusercontent.com/254612/156030727-252f5f44-2d0d-4f55-881a-4fc16c420802.png)

Reply:
I love that refreshed page design but 

>  I've got to say also a really big PR

I'll avoid it in this PR :)